.github/workflows/deploy-backend.yml

```
@@ -14,6 +14,7 @@ jobs:
         working-directory: ./Backend
     env:
       PROJECT_NAME: issue-tracker
+      JASYPT_KEY: issue-tracker-jasypt-key경
```

deploy 파일에 오타가 있었네요. ㅎㅎ

```suggestion
      JASYPT_KEY: issue-tracker-jasypt-key
```